src/AcmCountCard/AcmCountCard.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { ReactNode, useState } from 'react'
+import {
+    Card,
+    CardHeader,
+    CardHeaderMain,
+    CardActions,
+    CardTitle,
+    CardBody,
+    CardFooter,
+    CardProps,
+    Dropdown,
+    DropdownItem,
+    KebabToggle,
+    Skeleton,
+    DropdownItemProps,
+    CardActionsProps,
+} from '@patternfly/react-core'
+import { AcmTemplateIcon } from '../AcmIcons/AcmIcons'
+import { makeStyles } from '@material-ui/styles'
+
+type CardHeaderActions = DropdownItemProps & {
+    text: string
+}
+
+interface CardHeaderProps {
+    title: string
+    description: string
+    actions?: CardHeaderActions[]
+    onActionClick: (event: React.SyntheticEvent) => void
+    hasIcon?: boolean
+}
+
+interface CardFooterProps {
+    count: number
+    countDescription?: string
+    countLink?: string | ReactNode
+}
+
+interface CardDropdownProps {
+    dropdownItems: { text: string }[]
+    toggle?: React.ReactNode
+    onSelect: (event: React.SyntheticEvent) => void
+}
+
+type AcmCountCardProps = CardProps & {
+    id?: string
+    label?: string
+    loading?: boolean
+    onCardClick?: () => void
+    cardHeader?: CardHeaderProps
+    cardFooter?: CardFooterProps
+    count?: number
+    countTitle?: string
+    isFlat?: boolean
+}
+
+type SkeletonCard = CardProps & {
+    id?: string
+}
+
+const useStyles = makeStyles({
+    card: {
+        height: '250px',
+    },
+    headerDescription: {
+        fontSize: 'var(--pf-global--FontSize--sm)',
+        color: 'var(--pf-global--palette--black-700)',
+    },
+    count: {
+        color: (props: AcmCountCardProps) =>
+            props.count !== 0 ? 'var(--pf-global--link--Color)' : 'var(--pf-global--palette--black-900)',
+        fontSize: 'var(--pf-global--FontSize--3xl)',
+    },
+    countTitle: {
+        fontSize: 'var(--pf-global--FontSize--sm)',
+        fontWeight: 700,
+    },
+})
+
+export function CardDropdown(props: CardDropdownProps & CardActionsProps) {
+    const useStyles = makeStyles({
+        dropdown: {
+            width: '10rem',
+        },
+    })
+    const classes = useStyles()
+    const [isOpen, setOpen] = useState<boolean>(false)
+    const actionSelect = (event: React.SyntheticEvent) => {
+        // event?.stopPropagation()
+        setOpen(!isOpen)
+        props.onSelect(event)
+    }
+
+    return (
+        <Dropdown
+            className="dropdownMenu"
+            onClick={(e) => {
+                actionSelect(e)
+            }}
```

Comment:
This should be the same in a single line.
```suggestion
            onClick={actionSelect}
```